src/backend/Comments.mo

```diff
@@ -225,4 +233,46 @@ module {
         );
         List.toArray(comments);
     };
+
+    
+    //check principal is admin or not
+    func isAdmin(p : Principal) :  Bool {
+        if(Principal.equal(p , Principal.fromText(ADMIN[0]))){
+            true
+        }else if(Principal.equal(p , Principal.fromText(ADMIN[1]))){
+            true
+        }else {
+            return false
+        }
```

Comment:
should check all principals with an iter

Reply:
puted in utils and made it dynamic

Reply:
yeah i checked all pricipal using Array.vals()